test/signhash.ts

```diff
@@ -12,11 +13,31 @@ contract('SignHash', accounts => {
 
   describe('#sign', () => {
     it('should add sender to the list of signers', async () => {
-      await instance.sign(HASH);
+      await instance.sign(HASH, { from: accounts[0] });
 
       const signers = await instance.getSigners(HASH);
       assert.equal(signers.length, 1);
-      assert.equal(signers[0], ACCOUNT);
+      assert.equal(signers[0], accounts[0]);
+    });
+
+    it('should add multiple senders to the list of signers', async () => {
+      const count = 5;
+      for (let i = 0; i < count; i++) {
+        await instance.sign(HASH, { from: accounts[i] });
+      }
```

Comment:
No need to send them synchronously:

```
await Promise.all(
  [0..count].map(i => instance.sign(HASH, { from: accounts[i] })
);
```

Reply:
Fixed. I like this functional style but I needed to use `ramda` to support this. However, to be honest it made this test case more complicated. Now I need to check whether array contains signer, I cannot assume about its order. Check it out, it turned out pretty neat.